src/clj/rems/db/test_data.clj

```diff
@@ -87,43 +88,47 @@
-        start-date (db/create-form-item! {:type "date" :optional true :user owner :value 0})
-        expired (db/create-form-item! {:type "text" :optional true :user owner :value 0})
-        plan (db/create-form-item! {:type "attachment" :optional true :user owner :value 0})
-        maxlength-text (db/create-form-item! {:type "text" :optional false :user owner :value 0})
-        maxlength-texta (db/create-form-item! {:type "texta" :optional false :user owner :value 0})]
-    (db/end-form-item! {:id (:id expired)})
-    ;; link out of order for less predictable row ids
-    (db/link-form-item! {:form (:id form) :itemorder 1 :optional false :item (:id name) :user owner})
-    (db/link-form-item! {:form (:id form) :itemorder 3 :optional false :item (:id purpose) :user owner})
-    (db/link-form-item! {:form (:id form) :itemorder 2 :optional true :item (:id start-date) :user owner})
-    (db/link-form-item! {:form (:id form) :itemorder 4 :optional true :item (:id plan) :user owner})
-    (db/link-form-item! {:form (:id form) :itemorder 5 :optional true :item (:id maxlength-text) :user owner :maxlength 10})
-    (db/link-form-item! {:form (:id form) :itemorder 6 :optional true :item (:id maxlength-texta) :user owner :maxlength 100})
-    ;; localize
-    (db/localize-form-item! {:item (:id name) :langcode "fi" :title "Projektin nimi" :inputprompt "Projekti"})
-    (db/localize-form-item! {:item (:id name) :langcode "en" :title "Project name" :inputprompt "Project"})
-    (db/localize-form-item! {:item (:id purpose) :langcode "fi"
-                             :title "Projektin tarkoitus"
-                             :inputprompt "Projektin tarkoitus on ..."})
-    (db/localize-form-item! {:item (:id purpose) :langcode "en"
-                             :title "Purpose of the project"
-                             :inputprompt "The purpose of the project is to ..."})
-    (db/localize-form-item! {:item (:id start-date) :langcode "fi" :title "Projektin aloituspäivä" :inputprompt ""})
-    (db/localize-form-item! {:item (:id start-date) :langcode "en" :title "Start date of the project" :inputprompt ""})
-    (db/localize-form-item! {:item (:id expired) :langcode "en" :title "Expired form item" :inputprompt ""})
-    (db/localize-form-item! {:item (:id plan) :langcode "fi" :title "Projektisuunnitelma" :inputprompt ""})
-    (db/localize-form-item! {:item (:id plan) :langcode "en" :title "Project plan" :inputprompt ""})
-    (db/localize-form-item! {:item (:id maxlength-text) :langcode "fi" :title "Projektin lyhenne" :inputprompt ""})
-    (db/localize-form-item! {:item (:id maxlength-text) :langcode "en" :title "Project acronym" :inputprompt ""})
-    (db/localize-form-item! {:item (:id maxlength-texta) :langcode "fi" :title "Tutkimussuunnitelma" :inputprompt ""})
-    (db/localize-form-item! {:item (:id maxlength-texta) :langcode "en" :title "Research plan" :inputprompt ""})
+  (binding [context/*user* {"eppn" (users :owner)}]
```

Comment:
I think the user should be moved away from the context variable like in the application APIs also here. Maybe we can just add a task or TODO for it.